iOS/README.md

```diff
@@ -0,0 +1,20 @@
+# EnerTalk Sample App for iOS (Swift)
+
+
+Documentation link: http://developer.encoredtech.com/ios-sample/
+
+Tutorial video link: http://developer.encoredtech.com/get-started/
+
+
+Note
+====
+Before run this project, must be import our SDK frameworks - ETUserKit, ETPublicAPIKit, ETUICardKit.
```

Comment:
Before running this project, please import Enertalk SDK frameworks first -  ETUserKit, ETPublicAPIKit, ETUICardKit.